packages/server/modules/index.ts

```diff
@@ -77,7 +78,8 @@ const getEnabledModuleNames = () => {
     'serverinvites',
     'stats',
     'webhooks',
-    'workspacesCore'
+    'workspacesCore',
+    'multiregion'
```

Comment:
As mentioned before, the module itself is enabled always (for simplicity), but then functionality checks inside of it can use the feature flag